dotcom-rendering/src/web/lib/getUser.ts

```diff
@@ -19,7 +19,7 @@ const callApi = (url: string) => {
 };
 
 export const getUser = async (ajaxUrl: string): Promise<UserProfile | void> => {
-	const url = joinUrl([ajaxUrl, 'profile/me']);
+	const url = joinUrl([ajaxUrl, 'profile/me?strict_sanctions_check=false']);
```

Comment:
Ideally we should only perform the less strict sanctions check when we load the page, would it be correct to hard code it here ? We could also add a boolean parameter to this getUser function that defaults to true so that we are sure to only perform the less strict check when we are sure it's ok to do so

Reply:
There's a [separate thread of work](https://docs.google.com/document/d/1kSrZZrJ8hSw6bek5hGa9onE8QDz5TRHFqHV52EOFjts/edit#heading=h.gsod33a0pc47) looking at how we can refactor the way we load data into the page and this information is really useful for that, thanks for pointing it out!

I think in this case using a hard coded url is fine. It also means we won't split the cache by having different url params. If we need more control at a later stage, we can look at bringing in a different api for `getUser`

Reply:
If I understand correctly this api call happens on every article load, even if there's no discussion enabled.  I think a more complete fix to reduce the usage of the api would be to also stop making this discussion api call if we are not going to use the response